core/templates/pages/library-page/search-bar/search-bar.component.html

```diff
@@ -99,36 +97,36 @@
           </a>
-          <a (click)="toggleSelection('languageCodes', item.id)" *ngIf="last" (keydown)="onMenuKeypress($event, 'language', {tab: ACTION_CLOSE})"
+          <a tabindex="0" [attr.aria-label]="item.text + 'selected'" (keydown.enter)="toggleSelection('languageCodes', item.id)" (click)="toggleSelection('languageCodes', item.id)" *ngIf="last" (keydown)="onMenuKeypress($event, 'language', {tab: ACTION_CLOSE})"
              class="dropdown-item">
             <span class="e2e-test-selected">{{ item.text | translate }}</span>
             <i *ngIf="selectionDetails.languageCodes.selections[item.id]" class="fas fa-check float-right oppia-search-bar-category-selection-symbol"></i>
           </a>
         </li>
       </ng-container>
       <li ngbDropdownItem *ngIf="selectionDetails.languageCodes.numSelections > 1">
-        <a (click)="deselectAll('languageCodes')" class="dropdown-item"><i>Deselect All</i></a>
+        <a tabindex="0" (keydown.enter)="deselectAll('languageCodes')" (click)="deselectAll('languageCodes')" class="dropdown-item"><i>Deselect All</i></a>
       </li>
       <hr *ngIf="selectionDetails.languageCodes.numSelections > 0" class="oppia-search-bar-hr">
       <ng-container *ngFor="let item of selectionDetails.languageCodes.masterList">
         <li ngbDropdownItem *ngIf="!selectionDetails.languageCodes.selections[item.id]">
-          <a (click)="toggleSelection('languageCodes', item.id)" *ngIf="!first && !last" class="dropdown-item">
+          <a (click)="toggleSelection('languageCodes', item.id)" (keydown.enter)="toggleSelection('languageCodes', item.id)" tabindex="0" *ngIf="!first && !last" class="dropdown-item">
             <span class="e2e-test-deselected">{{ item.text | translate }}</span>
             <i *ngIf="selectionDetails.languageCodes.selections[item.id]" class="fas fa-check float-right oppia-search-bar-category-selection-symbol"></i>
           </a>
-          <a (click)="toggleSelection('languageCodes', item.id)" *ngIf="first" (keydown)="onMenuKeypress($event, 'language', {shiftTab: ACTION_CLOSE})" class="dropdown-item">
+          <a tabindex="0" (keydown.enter)="toggleSelection('languageCodes', item.id)" (click)="toggleSelection('languageCodes', item.id)" *ngIf="first" (keydown)="onMenuKeypress($event, 'language', {shiftTab: ACTION_CLOSE})" class="dropdown-item">
             <span class="e2e-test-deselected">{{ item.text | translate }}</span>
             <i *ngIf="selectionDetails.languageCodes.selections[item.id]" class="fas fa-check float-right oppia-search-bar-category-selection-symbol"></i>
           </a>
-          <a (click)="toggleSelection('languageCodes', item.id)" *ngIf="last" (keydown)="onMenuKeypress($event, 'language', {tab: ACTION_CLOSE})"
+          <a tabindex="0" (keydown.enter)="toggleSelection('languageCodes', item.id)" (click)="toggleSelection('languageCodes', item.id)" *ngIf="last" (keydown)="onMenuKeypress($event, 'language', {tab: ACTION_CLOSE})"
```

Comment:
Ditto

Reply:
Not done? (break this line onto multiple lines.

Reply:
Done